app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,16 @@
+<p>
+  <%= post.like.all.count %></br>
+  <% if post.like.all.where(user: current_user).length===0 %>
```

Comment:
This is currently erroring. It should be `post.likes`, not `post.like`. 

Also, you could write this line as `post.like.where(user: current_user).none?` for a somewhat more efficient query.

Reply:
Huh, this didn't error on my local host, but I can see your point! Pretty sure there isn't a git discrepancy but I'll investigate for curiosity.

Reply:
Oh, and thanks for the `.none?` tip, don't think I've used that before.

Reply:
Sorry, I must have deleted the S at some point later in the night after I confirmed it was working, I do get an error now.